projects/ngx-clipboard/src/lib/ngx-clipboard.service.ts

```diff
@@ -147,6 +147,10 @@ export class ClipboardService {
         // Prevent zooming on iOS
         ta.style.fontSize = '12pt';
         // Reset box model
+        ta.style.height = '1px';
+        ta.style.width = '1px';
+        ta.style.overflow = 'hidden';
+        ta.style.clip = 'rect(0 0 0 0);';
```

Comment:
in case you're wondering: https://tailwindcss.com/docs/screen-readers#:~:text=Properties-,sr%2Donly,-position%3A%20absolute